addons/content-index/src/main/java/org/commonjava/indy/content/index/ContentIndexObserver.java

```diff
@@ -272,8 +272,8 @@ private void propagatePathlessStoreEvent( Iterable<ArtifactStore> stores )
                 final StoreKey key = store.getKey();
 
                 Set<String> paths = new HashSet<>();
-                indexManager.removeAllIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPath());} );
-                indexManager.removeAllOriginIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPath());} );
+                indexManager.removeAllIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPathHash());} );
+                indexManager.removeAllOriginIndexedPathsForStore( key, indexedStorePath ->{paths.add(indexedStorePath.getPathHash());} );
 
```

Comment:
We should re-evaluate this.This is problematic because paths contain only hashes, while we need actual paths below to get SpecialPathInfo and the hash is not convertible to the full path.